kolibri/plugins/learn/assets/src/routes/index.js

```diff
@@ -98,10 +98,6 @@ export default [
     name: PageNames.LIBRARY,
     path: '/library' + optionalDeviceIdPathSegment,
     handler: to => {
-      if (!get(channels) || !get(channels).length) {
```

Comment:
I think we may actually want to keep this, but only in the case that the user is not signed in. As guest users cannot browse other libraries/devices, it seems worthwhile to keep the `CONTENT_UNAVAILABLE` redirect specifically for them.

Reply:
This has been resolved!